ecs-web/main.tf

```diff
@@ -62,8 +66,8 @@ EOF
 
 data "template_file" "concourse_basic_auth" {
   template = <<EOF
-{ "name": "CONCOURSE_BASIC_AUTH_USERNAME", "value": "$${concourse_auth_username}" },
-{ "name": "CONCOURSE_BASIC_AUTH_PASSWORD", "value": "$${concourse_auth_password}" },
+{ "name": "CONCOURSE_ADD_LOCAL_USER", "value": "$${concourse_auth_username}:$${concourse_auth_password}" },
+{ "name": "CONCOURSE_MAIN_TEAM_LOCAL_USER", "value": "$${concourse_auth_username}" },
```

Comment:
I think we don't need the local user to be added in the main team actually. This will only be needed for the `concourse-pipeline` resource on the customer team